RecoTracker/MeasurementDet/plugins/TkPixelMeasurementDet.cc

```diff
@@ -153,19 +153,24 @@ bool TkPixelMeasurementDet::hasBadComponents(const TrajectoryStateOnSurface& tso
     return false;
 
   auto lp = tsos.localPosition();
-  auto le = tsos.localError().positionError();
+  float xx = 0.;
+  float yy = 0.;
+  if (tsos.hasError()) {
+    xx = tsos.localError().positionError().xx();
+    yy = tsos.localError().positionError().yy();
+  }
```

Comment:
```suggestion
  float xx = tsos.hasError() ? tsos.localError().positionError().xx() : 0.0f;
  float yy = tsos.hasError() ? tsos.localError().positionError().yy() : 0.0f;
```

is slightly more compact.